engine/execution/ingestion/engine.go

```diff
@@ -310,68 +200,6 @@ func (e *Engine) reloadUnexecutedBlocks() error {
 		blockByCollection *stdmap.BlockByCollectionBackdata,
 		executionQueues *stdmap.QueuesBackdata,
 	) error {
-
-		// saving an executed block is currently not transactional, so it's possible
```

Comment:
this section of code was removed. is it not needed anymore?

Reply:
Good catch. 

Actually, the code is in `BlockLoader.LoadUnexecuted` method, I just need to copy the comments over